catalogue/aarch64-VMSA/README.md

```diff
@@ -0,0 +1,72 @@
+aarch64-VMSA: Catalogue with the semantics of Armv8-A VMSA
+============================================================
+
+Includes tests that exercise the semantics of the Virtual Memory
+System Architecture. Most of the tests in this catalogue, concurrently
+update and use Translation Table Descriptors.
+
+Simulating with herd7
+---------------------
+
+    % herd7 -variant vmsa,fatal -kinds tests/VMSA-kinds.txt tests/@all
+
+Running with litmus7
+--------------------
+
+VMSA litmus tests need to control the Translation Table Descriptors as
+well as exceptions and exception handlers. Consecuently, litmus7 needs
+to generate binaries that run in EL1 using hardware
+virtualization. `-mach kvm-aarch64` instructs litmus7 to generate such
```

Comment:
Does this benefit from mentioning `-mach kvm-armv8.1` for the tests with atomics? (Not sure off the top of my head if there are such tests in the VMSA catalogue.)

Reply:
Yes, this is a fair comment. There are a few tests here and in other catalogues that expect FEAT_LSE2. If one has HW implementing armv8.1, the easiest thing would be to ask them to use `-mach kvm-armv8.1`. If they don't I guess we have to create another list of tests (not `@all`). It would be nice if litmus7 could do something similar to TTHM; it's absence is detected at runtime and the execution exits without raising an error.